R/cql-geom-predicates.R

```diff
@@ -128,6 +118,58 @@ sf_text <- function(x) {
   sf::st_as_text(x)
 }
 
+
+#' Check spatial objects for WFS spatial operations
+#'
+#' Check a spatial object to see if it exceeds the current set value of
+#' 'bcdata.max_geom_pred_size' option. If the object does exceed the size
+#' threshold a bounding box was drawn around the polygon and all features
+#' within the box will be returned. Further options include:
+#' - Try adjusting the value of the 'bcdata.max_geom_pred_size' option
+#' - Simplify the spatial object to reduce its size
+#' - Further processing on the returned object
```

Comment:
I think it's a good idea to also refer to the vignette as suggested in the review

Reply:
Via a link to the pkgdown site or just a general reference?

Reply:
Maybe pkgdown? Probably good to try to push people there as much as possible